### DISCUSSION TITLE
btConvexHullComputer can fail with certain input

### DISCUSSION THREAD

**Discussion by erwincoumans** [bulletphysics/bullet3#3468] [General]
(copied from https://code.google.com/p/bullet/issues/detail?id=275)

The algorithm gives wrong results when I test it with the following vertices:

btConvexHullComputer\* convexhc = new btConvexHullComputer();
btAlignedObjectArray<btVector3> vertices;
vertices.resize(8);

vertices[0] = btVector3(2.0f, 3.535533905932738f, 3.535533905932737f);
vertices[1] = btVector3(4.0f, 2.0f, 0.0f);
vertices[2] = btVector3(0.0f, 2.0f, 0.0f);
vertices[3] = btVector3(1.0f, 0.0f, 0.0f);
vertices[4] = btVector3(4.0f, 1.414213562373095f, 1.414213562373095f);
vertices[5] = btVector3(0.0f, 1.414213562373095f, 1.414213562373095f);
vertices[6] = btVector3(3.0f, 0.0f, 0.0f);
vertices[7] = btVector3(2.0f, 5.0f, 0.0f);

**Comment by mcenderdragon** [comment]
does this still happen ?
